packages/asset-buyer/src/constants.ts

```diff
@@ -1,40 +1,35 @@
 import { SignedOrder } from '@0x/types';
 import { BigNumber } from '@0x/utils';
 
-import { AssetBuyerOpts, BuyQuoteExecutionOpts, BuyQuoteRequestOpts, OrdersAndFillableAmounts } from './types';
+import { AssetSwapQuoterOpts, ForwarderSwapQuoteExecutionOpts, OrdersAndFillableAmounts, SwapQuoteRequestOpts, SwapQuoteExecutionOpts } from './types';
 
 const NULL_ADDRESS = '0x0000000000000000000000000000000000000000';
 const MAINNET_NETWORK_ID = 1;
 
-const DEFAULT_ASSET_BUYER_OPTS: AssetBuyerOpts = {
+const DEFAULT_ASSET_SWAP_QUOTER_OPTS: AssetSwapQuoterOpts = {
```

Comment:
If these are passed into the methods that are part of the exported interface, these types should also be exported. Additionally, each of the fields should have a corresponding comment above this declaration (these will show up in the docs page).